lib/salus/scanners/report_ruby_gems.rb

```diff
@@ -76,8 +80,50 @@ def record_ruby_gem(name:, version:, source:, dependency_file:)
         type: 'gem',
         name: name,
         version: version,
-        source: source
+        source: source,
+        licenses: find_licenses_for(name, version)
       )
     end
+
+    def find_licenses_for(gem_name, version)
+      res = Net::HTTP.get_response(ruby_gems_uri_for(gem_name, version))
+
+      return [] unless res.is_a?(Net::HTTPSuccess)
+
+      licenses = JSON.parse(res.body)['licenses']
+      licenses.map { |license| spdx_license_for(license) }
```

Comment:
Are licenses from the json already in spdx format?
If not, do you know roughly what percentage of dice coefficient conversions are accurate?

Reply:
https://guides.rubygems.org/rubygems-org-api-v2/ does not mention that it's strictly following spdx format. In that case, it is better to use dice coefficient to match the license.

According to [this](https://towardsdatascience.com/metrics-to-evaluate-your-semantic-segmentation-model-6bcb99639aa2), dice coefficient is highly used for segmentation matching. I have not found any article/writing about mathematical analysis on dice coefficient. Another name of dice coefficient is [F-score](https://en.wikipedia.org/wiki/F-score). We can talk about them if you want.